src/ctia/store.clj

```diff
@@ -5,81 +5,79 @@
   (create-actor [this new-actor])
   (update-actor [this id actor])
   (delete-actor [this id])
-  (list-actors [this filtermap]))
+  (list-actors [this filtermap params]))
```

Comment:
Is it possible to strongly define the contents of the params map, using a schema, or perhaps a destructuring keyword spec?  I would at least want the expected contents of params to be documented in docstrings in the store protocol definitions.